numpy/core/src/umath/ufunc_object.c

```diff
@@ -5572,7 +5572,7 @@ ufunc_at(PyUFuncObject *ufunc, PyObject *args)
 
     PyUFuncGenericFunction innerloop;
     void *innerloopdata;
-    int i;
+    npy_intp i;
```

Comment:
Not all that surprised by this bug, given the variable is not used until 200 lines later. I suppose we can't use C99 declarations yet, since this will be backported.